src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCFUnitTest.java

```diff
@@ -0,0 +1,96 @@
+package org.broadinstitute.hellbender.tools.walkers.variantutils;
+
+import htsjdk.variant.variantcontext.*;
+import htsjdk.variant.vcf.VCFConstants;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ReblockGVCFUnitTest {
+    private final ReblockGVCF reblocker = new ReblockGVCF();
```

Comment:
I would just create this in each test that uses it, otherwise we potentially get weird stateful issues across multiple tests.